applications/nrf_desktop/doc/dvfs.rst

```diff
@@ -0,0 +1,104 @@
+.. _nrf_desktop_dvfs:
+
+DVFS module
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Desktop Dynamic Voltage and Frequency Scaling (DVFS) module is responsible for switching the frequency and voltage according to the application needs.
+DVFS is critical for balancing performance and power consumption, especially in battery-operated devices.
+
+In the nRF Desktop, the DVFS module enables the system to use the lowest possible frequency and voltage when the application performs less computation-challenging activities such as Bluetooth LE advertising.
+However, if performing more computation-consuming tasks nRF Desktop DVFS module will dynamically change to a higher frequency to increase performance.
```

Comment:
Additional suggestion - maybe something along these lines for better flow (incorporates Pekka's suggestion)?
```suggestion
However, for more compute-intensive tasks, the nRF Desktop DVFS module will dynamically switch to a higher frequency to enhance performance.
```